tests/test_functions

```diff
@@ -93,6 +93,13 @@ function test_install_nodejs {
     asserts_file_exist /usr/bin/npm
 }
 
+# test_install_python() - Verify the correct installation of Python
+function test_install_python {
+    install_python
+    asserts_installed_package python2.7
+    asserts_file_exist /usr/bin/python
```

Comment:
The line 100 should validate the tox binary, given that this is not installed via apt-get